anafora/copy_text.py

```diff
@@ -10,17 +10,22 @@
 
 def copy_timeml_text(text_dir, anafora_dir, xml_name_regex, write_dct):
     text_name_to_path = {}
+    text_name_to_folder = {}
     for dir_path, _, file_names in os.walk(text_dir):
         for file_name in file_names:
             if file_name.endswith(".tml"):
                 file_path = os.path.join(dir_path, file_name)
                 text_name_to_path[file_name[:-4]] = file_path
+                file_folder = dir_path.replace(text_dir, '')
```

Comment:
Does this work if `text_dir` does not end in a `/`? Maybe use `os.path.relpath` instead? (Same comment for the similar lines below).

Reply:
Changed!